Sources/StreamVideo/WebRTC/v2/SFU/Extensions/SFUSignalService.swift

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Combine
+import Foundation
+
+final class SFUSignalService {
```

Comment:
This is strange, we have 2 of these `SFUSignalService` it seems?

Reply:
No we don't. Why do you think that?